src/ontology/mondo-edit.obo

```diff
@@ -143769,12 +143769,7 @@ xref: OMIM:108770 {source="MONDO:equivalentTo"}
 xref: Orphanet:1344 {source="OMIM:108770"}
 xref: UMLS:C4551959 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1646392"}
 is_a: MONDO:0015281 {source="DC-OMIM:108770", source="MONDO:Redundant"} ! atrial standstill
-intersection_of: MONDO:0015281 ! atrial standstill
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10593 ! SCN5A
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4279 ! GJA5
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10593 {source="OMIM:108770"} ! SCN5A
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4279 {source="MONDO:mim2gene_medgen", source="OMIM:108770"} ! GJA5
```

Comment:
@matentzn since these gene associations do not exist in `master`, do you know how the OMIM pipeline added these in?

Reply:
If tehy are not in master, it means this branch is behind master and needs to be rebased.

Reply:
I did a local test last night with a fresh pull from mondo `master` and created a new branch and ran the OMIM gene update as `sh run.sh make update-omim-genes -B` and the same thing happened.

![Screenshot 2025-01-30 at 8 07 49 AM](https://github.com/user-attachments/assets/ef1a4029-6cfc-47ae-a71b-42cb7e10e1ff)

Reply:
This term has the annotation
`excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql`

These gene annotations should not be removed.

Reply:
@sabrinatoro can you remind me what your expectation is for entries that you said should be in the [exclusion file?](https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv)? I thought these were Mondo classes where OMIM source provenance should not be added and therefore the gene association(s) should not be on the class.

Reply:
> I did a local test last night with a fresh pull from mondo master and created a new branch and ran the OMIM gene update as sh run.sh make update-omim-genes -B and the same thing happened.

This image looks what I would expect... Nothing is removed, only evidence (on your screenshot)

Reply:
I don't know what the "exclusion file" is. 

This (and the others below) is a disease that must have 2 genes, and was reviewed by a curator.
The curator added the "excluded from QC check" to indicate that 
(1) it had been reviewed
(2) these multiple gene associations are correct
(3) these gene associations should be ignored during the gene-association update workflow. 

Please note that I do not know how (3) is handled.

Reply:
@matentzn you mentioned in the [first review](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1929062866) "I thought we decided not to preserve gene references if there is no provenance at all? This goes for various examples in this PR."

Reply:
@sabrinatoro the exclusion file is from the [spreadsheet ](https://github.com/monarch-initiative/mondo/pull/8108#pullrequestreview-2459808630) that you created and mentioned in the OMIM gene update in Nov. 

That PR does not have any gene associations for MONDO:0015281 'atrial standstill'.

Reply:
> That PR does not have any gene associations for MONDO:0015281 'atrial standstill'.

@twhetzel I don't follow... 
This PR show that the gene annotation (2 genes) are removed from atrial standstill 1 - MONDO:0007171 
This should not happen. 
Based on the notes from November "I also started this [spreadsheet](https://docs.google.com/spreadsheets/d/1akfYFkXLJJqEIGx7So2UMCmvnyZQSQj-0oTVTP5JHm4/edit?gid=0#gid=0) to keep track of the record that should be excluded from the pipeline because they do not fit the rules."
To me, it means that nothing should happen to this term, ie no gene removed, no gene added.

Reply:
I went back to look at _all_ the [commits](https://github.com/monarch-initiative/mondo/pull/8108/commits) from the previous PR and the last two you did and added the exclude from qc checks were not originally showing up and making this a very confusing picture. Now that I am aware of those later commits, I hope that clear things up 🙏

Reply:
Ok, I will manually add these back and Joe will update the OMIM ingest code to "protect" these so that nothing happens to this term, ie no gene removed, no gene added.